src/sentry/static/sentry/app/utils/highlightFuseMatches.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+const getFuseMatches = ({value, indices}) => {
+  let strLength = value.length;
+  let result = indices.reduce(
+    ({prev, text}, [start, end]) => {
+      // pop off the last stored value since we want to preserve string from prev.end to current start
+      text.pop();
```

Comment:
could this be at the end of function? slightly confused by it but trust it works.

Reply:
hate to say it but this might be better as a forEach with external mutation 😬 
I don't think it's that important for this to be readable as it's self contained but it's hard to keep track of the shape and meaning of all the values flying around